pages/goals.js

```diff
@@ -9,11 +9,15 @@ const goals = [
 ];
 
 function Goals() {
-  const { data: session } = useSession();
-  if (session) {
+  const { data: token } = useSession();
+  if (token) {
+    const accessToken = token.accessToken;
+    const payload = JSON.parse(Buffer.from(accessToken.split('.')[1], 'base64').toString());
+    const role = payload.realm_access.roles[1];
+    console.log(role);
     return (
       <Page title="SE: Goal App">
-        <GoalList goals={goals} />
+        <GoalList goals={goals} isEmployer={role==='Employer'} />
```

Comment:
Can you make 'Employer' role to be constant?. You can add something like 
export const USER_ROLE = {
employee: 'Employee',
employer: 'Employer'
}
then use role === USER_ROLE.employer

Reply:
we have helpers folder for constants,, you can add user.js file